.tekton/basic-component-pipeline.yaml

```diff
@@ -158,14 +135,18 @@ spec:
       value: $(params.clone-depth)
     - name: fetchTags
       value: $(params.clone-fetch-tags)
+    - name: ociStorage
+      value: $(params.output-image-repo):konflux-$(params.revision).git
+    - name: ociArtifactExpiresAfter
+      value: $(params.image-expires-after)
```

Comment:
I think that intermediate build artifacts don't need to be stored for a year. I even think they don't need to be stored for longer than a build because it is unlikely we'd look there often (if these were workspaces, we would not look there at all because we don't have access).

I suggest introducing a separate param for the lifetime of these intermediate artifacts independent of the lifetime for ultimate images. I suggest to set it to 13 weeks for the start.

Reply:
Done in https://github.com/stackrox/stackrox/pull/11616/commits/21469bc3ca7b0a706d1adbebf5ac316117177184, set to `1d`.